docs_app/src/index.html

```diff
@@ -151,6 +151,7 @@ <h2 style="color: red; text-align: center;">
       <b><i>This website requires JavaScript.</i></b>
     </h2>
   </noscript>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/rxjs/6.3.3/rxjs.umd.min.js"></script>
```

Comment:
This can be loaded asynchronously, to prevent it from blocking the main application loading. Please add the async attribute `<script src="" async></script>`. This will allow content loaded events to fire before that script is done loading.